app/assets/v2/js/pages/bounty_details2.js

```diff
@@ -300,7 +308,7 @@ Vue.mixin({
         case 'ETC':
           tenant = 'ETC';
           break;
-        
+
```

Comment:
@thelostone-mc

isn't this required?

```
case 'ONE':
  tenant = 'HARMONY';
  break;
```

Reply:
derp not sure how i missed that ! fixed !
thanks 🙌